### ISSUE TITLE
Call icons depend on color, give them other indicators?

### ISSUE THREAD

**Issue by ProMcTagonist** [qTox/qTox#823]
For the sake of colorblind users, and even normal ones (because there are tooltips right now), I think something like this would be nice.
![tox call icon indicators](https://cloud.githubusercontent.com/assets/4358079/5120908/4bbb74a8-703f-11e4-8937-110eb53d7067.png)
but way better of course

**Comment by suhr** [comment]
cast @ItsDuke

**Comment by ItsDuke** [comment]
Haha, this was done awhile ago, though I'd like to update it hopefully soon.

![1](https://cloud.githubusercontent.com/assets/7714365/5133799/16b112a8-70d6-11e4-8648-606b267bd26a.png)

**Comment by ghost** [comment]
@ItsDuke What's the point of the wave there?

**Comment by ProMcTagonist** [comment]
@ItsDuke Nice, clean, I like the microphone and speaker icons a lot. And the phone rotating to hangup position is clever. Shouldn't the video stop icon reflect that you're only going to stop broadcasting video? A stop sign might make people think it's going to stop everything.

**Comment by jvasile** [comment]
Red = "off" when it comes to the mic and speaker, but "on" when it comes to the phone and camera. So confusing.

Green = "go" in my mind when juxtaposed against red. But I have to turn the phone red to start a call.

**Comment by ItsDuke** [comment]
@mannol that was originally designed to show the waveform during audio calls. I've been told it's not very resource-heavy to have, but it's just an option.

@ProMcTagonist I've toyed with the idea of just putting just a slash through the video icon for when it's red, but I feel like that doesn't mesh well with the rotated phone visual. Though I might just scrap that and use the slash through any icon to be the universal for "off"

@jvasile I can definitely see where you're going with this. Do you feel like it would be more intuitive if the phone was something like this?

![1](https://cloud.githubusercontent.com/assets/7714365/5153238/131fec74-71ec-11e4-898f-226e4d8b121a.gif)

![untitled-1](https://cloud.githubusercontent.com/assets/7714365/5153239/16787c06-71ec-11e4-98f6-cdcd3ab227d0.gif)

**Comment by zetok** [comment]
@ItsDuke whoa, that's awesome.

Some user in groupchat suggested that one should have a way to manually tune audio I/O volume for each contact/groupchat separately.

I was thinking about sliders next to speaker/microphone buttons. Should they be always visible, or appear when hovered over mic/speaker button?

Also, about current design:
1. How it is supposed to indicate that you can't call contact (no support for a/v; /contact is offline)?
2. How one would be able to reject call?

**Comment by ItsDuke** [comment]
@zetok I agree that when in a groupchat, being able to mute/modulate the volume of each person would be super helpful.

Thinking off the top of my head about this, I would imagine that you could view a list of people in a group chat:

![1](https://cloud.githubusercontent.com/assets/7714365/5156875/8d45d666-72b5-11e4-9985-73183f31758a.PNG)

![2](https://cloud.githubusercontent.com/assets/7714365/5156878/8fbec1aa-72b5-11e4-93fc-5762305a2041.PNG)

This image is a little old, but you get the idea.

It would be best if you could hover over each user and mute/change the volume that way, but there's currently not enough space in that mockup to do it. It might be better if we simply swapped to showing their avatars and names in that list, which would allow a bit more room to work with when you hover over them.

And in response to your secondary questions:
1: I feel like simply hiding the button would be the simplest choice. There are other options, like greying it out or similar things, but I feel like simply hiding the button would be the least confusing method.

2: In another Tox Issue (https://github.com/tux3/qTox/issues/515) I posted this:

![6b18824c-63a0-11e4-9b2d-bc875be41a3d](https://cloud.githubusercontent.com/assets/7714365/5156897/6e70fc10-72b6-11e4-810e-2efbe8f137a0.gif)

This could potentially work as shown for a contact you are currently chatting with, or orient it to the side to show that a contact from your list is currently calling, so you can accept/reject without having to leave your current chat.

Just some food for thought.

**Comment by ProMcTagonist** [comment]
![screenshot - 03142015 - 11 21 42 pm](https://cloud.githubusercontent.com/assets/4358079/6654944/909fccd2-caa2-11e4-81be-97e17962d5ce.png)

Could we avoid the mute buttons' color issues like this?

**Comment by zetok** [comment]
@ProMcTagonist nice, although I think that line crossing mic/speaker should be red?

Edit: or maybe not..

**Comment by ProMcTagonist** [comment]
@zetok with Tox Red
![redline](https://cloud.githubusercontent.com/assets/4358079/6657774/f12ef9c2-cb12-11e4-84d6-4f7837471c21.png)

**Comment by dubslow** [comment]
@ItsDuke Can you make the primitive images available please? Those mockups are pretty neat looking, it would be nice to actually have them :)

**Comment by ItsDuke** [comment]
@dubslow I just pushed versions of these to my repository. Enjoy!

**Comment by ProMcTagonist** [comment]
@ItsDuke I think 

> ![this](https://cloud.githubusercontent.com/assets/7714365/5153238/131fec74-71ec-11e4-898f-226e4d8b121a.gif)
> 
> ![and this](https://cloud.githubusercontent.com/assets/7714365/5153239/16787c06-71ec-11e4-98f6-cdcd3ab227d0.gif)

are way too complicated and confusing.

I really do like the phone turning, though, and I like 

> ![this](https://cloud.githubusercontent.com/assets/7714365/5156897/6e70fc10-72b6-11e4-810e-2efbe8f137a0.gif)

too.

What do you think about 

> ![this](https://cloud.githubusercontent.com/assets/4358079/6657774/f12ef9c2-cb12-11e4-84d6-4f7837471c21.png)

?

**Comment by Getron** [comment]
The size of label doesn't match the text on other languages, for example, german:
![auswahl_254](https://cloud.githubusercontent.com/assets/5913218/15753960/41bb4340-28f6-11e6-8a43-c6f933fc46df.png)

Like you see, the icons after text are cuted.

My OS is Netrunner 2016.01 KDE Arch Rolling release, with QtCurve and dark arc theme

**Comment by zetok** [comment]
@Getron that's a separate issue.
